zio-config/src/main/scala/zio/config/PropertyType.scala

```diff
@@ -68,27 +68,4 @@ object PropertyType extends AttemptSyntax {
       override def write(a: Option[A]): String = a.map(aa => propertyType.write(aa)).getOrElse("")
       override def description: String         = s"option of ${propertyType.description}"
     }
-
-  private[config] def ofList[A](propertyType: PropertyType[A]): PropertyType[List[A]] = new PropertyType[List[A]] {
-    override def read(value: String): Either[ErrorType, List[A]] =
-      StringType.read(value) match {
-        case Right(v) => sequence(v.split(",").toList.map(t => propertyType.read(t)))
```

Comment:
Our library assumed users are going to put their list as comma separated strings.
Instead we provide better combinators, that allows the user to design the list of variables (in the environment) in any way that they need.